py3_ruby/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
 
 # skip installing gem documentation
 RUN echo 'gem: --no-rdoc --no-ri --no-document' >> "/etc/gemrc" && \
-  gem install bundler -v 1.17
+  gem install bundler -v --no-document 1.17.3
```

Comment:
Shouldn't the version number precede the `-v` option?

Reply:
you are right. flxed throughout